src/execution_plan/execution_plan.h

```diff
@@ -73,6 +73,22 @@ OpBase *ExecutionPlan_LocateReferences(OpBase *root, rax *references);
  * The Create operation should never introduce a new node 'a'. */
 void ExecutionPlan_BoundVariables(const OpBase *op, rax *modifiers);
 
+/* Allocate a new ExecutionPlan segment. */
+ExecutionPlan *ExecutionPlan_NewEmptyExecutionPlan(void);
+
+/* Build a tree of operations that performs all the worked required by the clauses of the current AST. */
+void ExecutionPlan_PopulateExecutionPlan(ExecutionPlan *plan, ResultSet *result_set);
+
+/* Build an array of const strings to populate the 'modifies' arrays of Argument ops.
+ * Use after the call to ExecutionPlan_BoundVariables. */
+const char **ExecutionPlan_BuildArgumentModifiesArray(rax *bound_vars);
```

Comment:
These functions are currently in `execution_plan.c`, not `execution_plan_modify.c`.

Reply:
fixed